vite.config.pages.ts

```diff
@@ -1,11 +1,19 @@
+import path from 'node:path';
+
 import react from '@vitejs/plugin-react';
+import wyw from '@wyw-in-js/vite';
 import { defineConfig } from 'vite';
 import svgr from 'vite-plugin-svgr';
-import path from 'node:path';
 
 export default defineConfig({
   plugins: [
     react(),
+    wyw({
+      include: ['**/*.{ts,tsx}'],
+      babelOptions: {
+        presets: ['@babel/preset-typescript', '@babel/preset-react', '@wyw-in-js/babel-preset'],
+      },
+    }),
```

Comment:
could you explain why we need this please?

Reply:
It is simply the configuration of the library. 
https://github.com/callstack/linaria/blob/master/docs/BUNDLERS_INTEGRATION.md#Vite

To understand how it works, I recommend studying the concept of Babel.

Reply:
ok but why do we need wyw-in-js library?

Reply:
Currently, we have installed and applied the Linaria library as an experiment (a zero-runtime CSS-in-JS library). 
One of the configurations needed in the Linaria library is `wyw-in-js`.

The role of `wyw-in-js` includes general tools necessary for developing CSS-in-JS libraries (such as JavaScript parsers for converting JS to CSS, and various plugins for different bundlers).

Since Linaria is developed using this `wyw-in-js` library, other configurations are also done through `wyw-in-js`, so it needs to be installed together.

<img width="872" alt="image" src="https://github.com/user-attachments/assets/be65c814-97bb-45b2-9839-cefbd261f3f9">

Reply:
Thanks for the explaination 👍